task/06-conditions-n-loops-tasks.js

```diff
@@ -431,7 +533,46 @@ function getMatrixProduct(m1, m2) {
  *
  */
 function evaluateTicTacToePosition(position) {
-    throw new Error('Not implemented');
+    let winCoords = [
+        [11,12,13], [21,22,23], [31,32,33],
+        [11,21,31], [12,22,32], [13,23,33],
+        [11,22,33], [13,22,31]
+    ];
+    let n = 3;
+
+    function transformMatrix(position, symb) {
+        let arr = [];
+        for(let i = 1; i <= n; i++){
+            for(let j = 1; j <= n; j++) {
+                winCoords[i][j] === arr[j];
+                if (position[i-1][j-1] === symb)
+                    arr.push(Number(i.toString() + j));
+            }
+        }
+        return arr;   
+    }
+
+    function isWin(arr, symb) {
+       for (let i = 0; i < winCoords.length; i++) {
+           let truth = 0;
+           for (let j = 0; j < n; j++) {
+                for (let k = 0; k < arr.length; k++ ) {
+                    if ( winCoords[i][j] === arr[k])
+                        truth += 1    
+                }
+           }
+           if (truth === 3) return symb; 
+       }
+    return false;    
+    }
+
+    let arr = transformMatrix(position, 'X');
```

Comment:
you don't really care if it's an X or O . Just go over the wining positions and make sure they have the same value and just  get one this values.